### ISSUE TITLE
No Status-Update in HA Group-Bound-Switch-Action

### ISSUE THREAD

**Issue by attkri** [zigbee2mqtt/hassio-zigbee2mqtt#73]
**Description of problem:**

I have organized my IKEA lights into groups and tied an IKEA switch to the group. When I switch the group via HA, the status of the lights and the group changes. However, if I switch via the switch, the light is switched but the status in HA does not change.

**Zigbee2mqtt add-on version (if edge, please report commit hash):**

Zigbee2mqtt add-on Current version: 1.18.1-2

**Running platform (details from Suypervisor > System)**

core-2021.3.4
supervisor-2021.03.6
Home Assistant OS 5.12

**Comment by ronniebee** [comment]
> **Description of problem:**
> 
> I have organized my IKEA lights into groups and tied an IKEA switch to the group. When I switch the group via HA, the status of the lights and the group changes. However, if I switch via the switch, the light is switched but the status in HA does not change.
> 
> **Zigbee2mqtt add-on version (if edge, please report commit hash):**
> 
> Zigbee2mqtt add-on Current version: 1.18.1-2
> 
> **Running platform (details from Suypervisor > System)**
> 
> core-2021.3.4
> supervisor-2021.03.6
> Home Assistant OS 5.12

@attkri How did you manage to bind the switch? see my post #71

**Comment by attkri** [comment]
@ronniebee 
Switch deleted (IKEA 1743) and paired again. Then under the **Bind** switch, unbind the **default_bind_group**. Target group selected, **OnOff** activated and then the button Bind becomes free and can be pressed. And always make sure that the IKEA switch is clicked to turn it on.

**Comment by attkri** [comment]
![image](https://user-images.githubusercontent.com/59642616/112180823-2d1fa480-8bfc-11eb-9342-b89b5947f76d.png)

![image](https://user-images.githubusercontent.com/59642616/112180979-49bbdc80-8bfc-11eb-9052-d519fa469149.png)

![image](https://user-images.githubusercontent.com/59642616/112181078-62c48d80-8bfc-11eb-9ce6-f41050e486f5.png)

**Comment by ronniebee** [comment]
> @ronniebee
> Switch deleted (IKEA 1743) and paired again. Then under the **Bind** switch, unbind the **default_bind_group**. Target group selected, **OnOff** activated and then the button Bind becomes free and can be pressed. And always make sure that the IKEA switch is clicked to turn it on.

Yes! that worked well... thx!

**Comment by filewalker1** [comment]
i can confirm your issues. I had the same and did not find any solution. So i had to rethink this solution and tried many things with automations, .... but nothing was even near the repsonse time when linked directly. I ended up installing ControlerX in HACS. This works flawless. The response time is very good, close to direct binding but with the drawback, that the light only works, if HA, Z2M, HACS + ControllerX are working. Direct Binding gives a bit more redundancy...

**Comment by attkri** [comment]
> i can confirm your issues. I had the same and did not find any solution. So i had to rethink this solution and tried many things with automations, .... but nothing was even near the repsonse time when linked directly. I ended up installing ControlerX in HACS. This works flawless. The response time is very good, close to direct binding but with the drawback, that the light only works, if HA, Z2M, HACS + ControllerX are working. Direct Binding gives a bit more redundancy...

Great, yet another AddOn. I will have a look at it and evaluate it for myself. But this chaos in home automation is the reason why this topic like IPv6 and PKI needs decades to arrive.

**Comment by attkri** [comment]
UPDATE: 

Due to a channel change I had to re-pair all devices.

1. I have divided lamps into groups via MQTT message.
2. bound the corresponding switch to the group via UI, for example: `Source EP: 1; Destination: Top Floor; Clusters: OnOff`. As well as the default binding is removed.

The switch does what it is supposed to do. It switches the lamps of the group on or off in a flash.

BUT: The state of the lamps is not transmitted to whenever. In HA the previous state remains. What disturbs, since the automations do not work correctly any more since the lamps show a wrong state.

Any suggestions?
